react/src/components/Weather/CurrentWeather/CurrentWeather.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import CurrentStyle from './CurrentW.module.css';
+import sun from '../../../icons/desktop/weathersun.png';
+import rain from '../../../icons/desktop/weatherrain.png';
+import cloud from '../../../icons/desktop/weathercloud.png';
+import { format } from 'date-fns';
+import { pl } from 'date-fns/locale'
+
+const icons = {
+    'Clear': sun,
+    'Rain': rain,
+    'Clouds': cloud
+};
```

Comment:
To jest jedna z tych stałych zmiennych, gdzie dobrą praktyką jest użycie takiej konwencji:

```suggestion
const ICONS = {
    'CLEAR': sun,
    'RAIN': rain,
    'CLOUDS': cloud
};
```

A później użycie np. `ICONS.CLEAR`. Taką praktykę stosuje się w `typescript` (ale nie z użyciem constów, w typescript mamy do tego `enum`), ale generalnie, jeśli zmienna się nigdy nie zmieni i jest reużywalna, to warto odróżnić taką zmienną drukowanymi literami. To samo moglibyśmy zastosować dla np. zmiennej (chyba takiej tu nie ma) `BUTTON_VARIANT`.

Reply:
icons zmienione na ICONS, ale te Clear/Rain/Clouds nie mogę (CHYBA) zmienię bo to w takiej wersji się wyęwietla w consoli=>porownuję=>bierze ikonke.